packages/bigtable/src/table.js

```diff
@@ -659,7 +677,17 @@ Table.prototype.getRows = function(options, callback) {
  *   }
  * ];
  *
- * table.insert(entries, function(err, apiResponse) {});
+ * table.insert(entries, callback);
+ *
+ * //-
+ * // A stream mode is also provided, this is useful when making a large number
```

Comment:
Okay, I think this being a stream is a bit confusing, since all it's for is reporting how successful the request was. Can this be an event emitter instead that only emits `error` and `complete` events, more like an Operation object?